lib/google_search_data_viewer_worker/keywords/helpers/job_creation.ex

```diff
@@ -0,0 +1,14 @@
+defmodule GoogleSearchDataViewerWorker.Keywords.JobCreationHelper do
+  alias GoogleSearchDataViewerWorker.Keywords.SearchWorker
+
+  def create_keyword_upload_jobs_with_delay(keyword_uploads, delay \\ 3) do
```

Comment:
Let's add test for this function

Reply:
By checking the jobs are inserted into the `oban_jobs` table with the delays?

Reply:
By checking the jobs are inserted into the table `oban_jobs` with the correct delay? I can't check that a list of Oban Jobs are created as the function also inserts them.

Reply:
I would not check the jobs are inserted with the corresponding delay in the `oban_jobs` table but that the returned list of jobs from `insert_all` matches?